package.json

```diff
@@ -90,5 +92,6 @@
         }
       ]
     }
-  }
+  },
+  "packageManager": "yarn@4.7.0"
```

Comment:
would you have any objection if we used pnpm instead? it seems to have more momentum these days, and less quirks or caveats for contributors. we can use `pnpm import` to set up the lockfile.

i know it looks a little awkward in the `package.json` but we should also set the `packageManager` field via `corepack use`, which adds the sha512 sum (unfortunately corepack doesn't otherwise have a proper pinning/integrity mechanism besides this).

Reply:
Sure, we can do that :)